spec/round_spec.rb

```diff
@@ -0,0 +1,125 @@
+      round.take_turn("Baton Rouge")
+
+      expect(round.percent_correct).to eq(50.0)
+    end
+
+
+  it 'creates a turn with the correct class' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+    new_turn = round.take_turn("Juneau")
+
+    expect(new_turn).to be_instance_of(Turn)
+  end
+
+  it 'can count the turns taken' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+    new_turn = round.take_turn("Juneau")
+
+    expect(round.turns.count).to eq(1)
+  end
+
+  it 'can count the turns taken' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("true or false? 5 < 5", false, :Math)
+    card_3 = Card.new("true or false? 5 < 10 ", true, :Math)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+    new_turn = round.take_turn("Juneau")
+
+    expect(round.turns.last.feedback).to eq("Correct!")
+  end
+
+  it 'can count the correct turns by category' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+    new_turn = round.take_turn("Juneau")
+
+    expect(round.number_correct_by_category(:Geography)).to eq(1)
+  end
+
+  it 'can count the percent correct by category' do
+    card_1 = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    card_2 = Card.new("The Viking spacecraft sent back to Earth photographs and reports about the surface of which planet?", "Mars", :STEM)
+    card_3 = Card.new("Describe in words the exact direction that is 697.5° clockwise from due north?", "North north west", :STEM)
+    cards = [card_1, card_2, card_3]
+    deck = Deck.new(cards)
+    round = Round.new(deck)
+    round.take_turn("Juneau")
+    round.take_turn("Venus")
+
+    expect(round.percent_correct_by_category(:Geography)).to eq(50.0)
```

Comment:
```
expected: 50.0
got: 100.0
```

I want to go out on a limb and say that the expectation for this test is incorrect. If you look at the setup, there are 3 cards, 1 is a Geography card, 2 are STEM cards. We answer the Geography question correctly on line 118, and your test is checking the percentage correct by category and passing `:Geography` which we got right, so the expectation you actually be `eq(100.0)` and not `eq(50.0)`

Reply:
Yep! I see that now. You are correct.